docs/providers/aws/README.md

```diff
@@ -93,6 +93,7 @@ If you have any questions, [search the forums](https://forum.serverless.com?utm_
         <li><a href="./events/schedule.md">Schedule</a></li>
         <li><a href="./events/sns.md">SNS</a></li>
         <li><a href="./events/sqs.md">SQS</a></li>
+        <li><a href="./evetns/alb.md">ALB</a></li>
```

Comment:
There's a typo here. Should be `./events/alb.md`.

Reply:
I submitted a [separate pull request](https://github.com/serverless/serverless/pull/6292) to fix the typo, since this one is already parked as Done.